tests/integration/data/test_buf

```diff
@@ -37,9 +37,8 @@ PROTO_VERS_TO_S_CLIENT_ARG = {
 S_CLIENT_SUCCESSFUL_OCSP="OCSP Response Status: successful"
 
 def get_supported_curves_list_by_version(libcrypto_version):
-   # Curve X25519 is supported for Openssl 1.1.0 and higher
     if libcrypto_version == "openssl-1.1.1":
-        return  ["P-256", "P-384", "X25519"]
```

Comment:
This file is being used as input for another test-- do we actually need to keep it up-to-date?

Reply:
If  we are using these variables in the test, then yes, at this point we aren't.

Reply:
From offline chat: This file seems to be random data to the integration test `s2n_dynamic_record_test.py`. This file needs to be cleaned-up.